src/x3f_image.c

```diff
@@ -98,6 +99,34 @@
   return 1;
 }
 
+static int x3f_get_fake_camf_matrix(x3f_t *x3f, char *name,
+				    x3f_area16_t *image, uint32_t *rect)
+{
+  uint32_t column[4];
+  int i;
+
+  if (!x3f_get_camf_matrix(x3f, "DarkShieldColRange", 2, 2, 0, M_UINT, column))
+    return 0;
+
+  if (!strcmp(name, "FakeDarkShieldLeft")) {
+    rect[0] = column[0];
+    rect[1] = 0;
+    rect[2] = column[1];
+    rect[3] = image->rows; /* TODO - is this right? */
+    return 1;
+  }
+
+  if (!strcmp(name, "FakeDarkShieldRight")) {
+    rect[0] = column[2];
+    rect[1] = 0;
+    rect[2] = column[3];
+    rect[3] = image->rows; /* TODO - is this right? */
```

Comment:
Both the min and the max? I.e. 1 and 3.
And - what about 0 and 2, the columns?
keep_area all the way? Or is the columns in the meta data already in keep_area coordinates?